packages/esm-patient-registration-app/src/patient-registration/patient-registration-hooks.ts

```diff
@@ -109,30 +109,33 @@ export function useInitialFormValues(patientUuid: string, isLocal: boolean): [Fo
   }, [isLoadingPatientToEdit, patientToEdit, patientUuid]);
 
   useEffect(() => {
+    const fieldConfigIdentifier = fieldConfigurations.identifier;
+    const fieldConfigPhoneUuid = fieldConfigurations.phone.personAttributeUuid;
+
     const fetchValues = async () => {
-      if (mpiPatient?.data?.identifier) {
-        const identifiers = await getIdentifierFieldValuesFromFhirPatient(
-          mpiPatient.data,
-          fieldConfigurations.identifier,
-        );
+      if (mpiPatient?.identifier) {
+        const identifiers = await getIdentifierFieldValuesFromFhirPatient(mpiPatient, fieldConfigIdentifier);
 
         const values = {
           ...initialFormValues,
-          ...getFormValuesFromFhirPatient(mpiPatient.data),
-          address: getAddressFieldValuesFromFhirPatient(mpiPatient.data),
+          ...getFormValuesFromFhirPatient(mpiPatient),
+          address: getAddressFieldValuesFromFhirPatient(mpiPatient),
           identifiers,
-          attributes: getPhonePersonAttributeValueFromFhirPatient(
-            mpiPatient.data,
-            fieldConfigurations.phone.personAttributeUuid,
-          ),
+          attributes: getPhonePersonAttributeValueFromFhirPatient(mpiPatient, fieldConfigPhoneUuid),
         };
 
         setInitialFormValues(values);
       }
     };
 
     fetchValues();
-  }, [mpiPatient, isLoadingMpiPatient]);
+  }, [
+    fieldConfigurations.identifier,
+    fieldConfigurations.phone.personAttributeUuid,
+    initialFormValues,
+    isLoadingMpiPatient,
+    mpiPatient,
```

Comment:
This adds missing dependencies to the effect hook. I don't know why the `exhaustive-deps` eslint rule is not enabled for this repo. It should be so we can catch subtle bugs related to violations of rules of hooks.